packages/headless/src/features/commerce/facets/facet-search-set/category/commerce-category-facet-search-request-builder.ts

```diff
@@ -34,7 +36,7 @@ export const buildCategoryFacetSearchRequest = async (
     clientId,
     context,
     ...restOfCommerceAPIRequest
-  } = buildCommerceAPIRequest(state, relay);
+  } = buildCommerceAPIRequest(state, relay, navigatorContext);
```

Comment:
Passing both `relay` and `navigatorContext` is good for now, and we can revisit in the future.

Have we logged a jira to take another look at this though, maybe in the context of headless commerce SSR?

Reply:
Headess Commerce SSR is to be considered for investigation in Q3. For now I'll add a link to the Epic (that I'll create now)

Reply:
Thanks,